src/main/java/com/powsybl/openloadflow/dc/equations/DcEquationSystemUpdater.java

```diff
@@ -33,19 +33,7 @@ private void updateElementEquations(LfElement element, boolean enable) {
                         .setActive(!enable);
             }
         } else {
-            // update all equations related to the element
-            for (var equation : equationSystem.getEquations(element.getType(), element.getNum())) {
-                if (equation.isActive() != enable) {
-                    equation.setActive(enable);
-                }
-            }
-
-            // update all equation terms related to the element
-            for (var equationTerm : equationSystem.getEquationTerms(element.getType(), element.getNum())) {
-                if (equationTerm.isActive() != enable) {
-                    equationTerm.setActive(enable);
-                }
-            }
+            nonBranchEquationsUpdate(element, enable);
```

Comment:
The first part is dedicated to updating zero impedance branch equations. The second part is for the rest, included impedant branch equations.

Reply:
Yes I figured the name would be problematic, I was interested in factorization for ci, do you have a suggestion ?